sections/04_study.md

```diff
@@ -222,13 +222,67 @@ systems are limited by expensive computational cost proportional to the number
 of training and testing sequences. More recently,
 [@tag:Alipanahi2015_predicting] showed that convolutional neural network
 models could achieve state of the art results on the TFBS task and are scalable
-to a large number of genomic sequences. [@tag:Lanchantin2016_motif] introduced
-several new convolutional and recurrent neural network models for predicting
-TFBSs, but it remains unclear which neural architectures work best for all
-samples and TFs. While neural architectures are rapidly changing and producing
+to a large number of genomic sequences. 
+
+Since the work by [@tag:Alipanahi2015_predicting] was published, there have
+been a multitude of deep learning works on the TFBS task. Due to the 
+motif-driven nature of the TFBS task, most architectures
+have been convolutional-based, as summarized in [@tag:Zeng2016_convolutional].
+[@tag:Lanchantin2016_motif] introduced several new convolutional and 
+recurrent neural network models for predicting TFBSs, which showed improvements
+on the baseline datasets. It is also important
+to note that DNA sequence tasks are fundamentally different than natural
+language tasks, and thus the models should be adapted from traditional
+deep learning models in order to account for such differences.
+For example, motifs may appear in either strand of a DNA sequence, resulting
+in two different forms of the motif (forward and reverse complement) due 
+to complementary base pairing.
+[@tag:Shrikumar2017_reversecomplement] created a convolutional model
+which can find motifs in both directions.
+
+
+Deep learning models have shown great accuracy on the TFBS task, but
+the results are not fully convincing for several reasons based on the
+datasets used and the evaluation of the model's predictions on such
+datasets. First, ChIP-seq experiments give a continuous value of binding 
+likelihood at a certain location based on many experiments at that
+location in DNA. Based on these values, the TFBS task is usually
+converted into a binary classification task based on a certain threshold
+of the ChIP-seq value. However, the value may not be accurate in itself.
+There is no guarantee of a binding or non binding at that location
+depending on which person's sequence it is. Second, most datasets predict
+one TF at a time, using a separate model for each TF. In reality, there
```

Comment:
Isn't a lot of work in this domain multi-task? Or are you referring to something else?